packages/blocks/src/page-block/edgeless/components/text/edgeless-group-title-editor.ts

```diff
@@ -0,0 +1,129 @@
+import { assertExists } from '@blocksuite/global/utils';
+import { ShadowlessElement, WithDisposable } from '@blocksuite/lit';
+import { html } from 'lit';
+import { customElement, property, query } from 'lit/decorators.js';
+import { styleMap } from 'lit/directives/style-map.js';
+
+import type { RichText } from '../../../../_common/components/rich-text/rich-text.js';
+import type { GroupElement } from '../../../../surface-block/index.js';
+import { Bound } from '../../../../surface-block/index.js';
+import type { EdgelessPageBlockComponent } from '../../edgeless-page-block.js';
+
+@customElement('edgeless-group-title-editor')
+export class EdgelessGroupTitleEditor extends WithDisposable(
+  ShadowlessElement
+) {
+  @query('rich-text')
+  richText!: RichText;
+
+  @property({ attribute: false })
+  group!: GroupElement;
+  @property({ attribute: false })
+  edgeless!: EdgelessPageBlockComponent;
+
+  get vEditor() {
+    assertExists(this.richText.vEditor);
+    return this.richText.vEditor;
+  }
+  get vEditorContainer() {
+    return this.vEditor.rootElement;
+  }
+
+  override async getUpdateComplete(): Promise<boolean> {
+    const result = await super.getUpdateComplete();
+    await this.richText?.updateComplete;
+    return result;
+  }
+
+  override firstUpdated(): void {
+    const dispatcher = this.edgeless.dispatcher;
+    const { surface } = this.edgeless;
+    assertExists(dispatcher);
+
+    this.updateComplete.then(() => {
+      this.vEditor.selectAll();
+
+      surface.updateElementLocalRecord(this.group.id, { titleHide: true });
```

Comment:
Use `showTitle` should be more straightforward. Avoid negative naming (and default field values) should be better, since it's better to use `if (show)` than `if (!hide)`, especially in complex condition nesting.

Also, the field naming grammar should be `adj. + n.`, instead of `n. v.`, which is semantically incorrect.